--- emsdk	2015-01-26 20:40:05.000000000 +0100
+++ emsdk.p	2015-04-01 14:00:56.752117623 +0200
@@ -1003,6 +1003,8 @@
       return t
   return None
 
+def patch_update(): os.system("sed -i 's/cpu_cores = .*/cpu_cores = 1/' emsdk")
+
 def update_emsdk():
   if WINDOWS:
     download_and_unzip(urlparse.urljoin(emsdk_packages_url, 'emsdk_windows_update.zip'), emsdk_path(), download_even_if_exists=True)
@@ -1010,6 +1012,7 @@
   elif OSX or LINUX:
     download_and_unzip(urlparse.urljoin(emsdk_packages_url, 'emsdk_unix_update.tar.gz'), emsdk_path(), download_even_if_exists=True)
     rmfile('zips/emsdk_unix_update.tar.gz')
+    patch_update()
   else:
     print 'Unsupported OS, cannot update!'
 
